packages/app/src/cli/models/extensions/specifications/payments_app_extension_schemas/redeemable_payments_app_extension_schema.ts

```diff
@@ -0,0 +1,55 @@
+import {BaseSchema} from '../../schemas.js'
+import {zod} from '@shopify/cli-kit/node/schema'
+
+export type RedeemablePaymentsAppExtensionConfigType = zod.infer<typeof RedeemablePaymentsAppExtensionSchema>
+
+const MAX_LABEL_SIZE = 50
```

Comment:
Why do we have this constant (with the same value) repeated in multiple schema files? e.g: [here](https://github.com/Shopify/cli/blob/98cf1ce9258ab9b4c739a58c76fd282a4738909b/packages/app/src/cli/models/extensions/specifications/payments_app_extension_schemas/offsite_payments_app_extension_schema.ts#L6)

Reply:
I think each label size can be tailored specifically to the extension type which is why I kept them in separate files even though they are technically the same value